packages/myst-frontmatter/src/frontmatter/validators.ts

```diff
@@ -370,14 +378,14 @@ export function validateThebe(input: any, opts: ValidationOptions) {
     output.binder = validateBooleanOrObject(
       value.binder,
       incrementOptions('binder', opts),
-      validateThebeBinderOptions,
+      validateBinderHubOptions,
     );
   }
   if (defined(value.server)) {
     output.server = validateBooleanOrObject(
```

Comment:
For these, I think we should be explicit in returning the object always, then the config is clear and there is less validation in the clients/theme.